README.md

```diff
@@ -30,6 +30,12 @@ OpenSearch and its components are built from source, assembled, signed and teste
 
 TODO
 
+#### Releasing for FreeBSD
+
+The FreeBSD ports and packages for OpenSearch are managed by an *[OpenSearch Team](https://wiki.freebsd.org/OpenSearch)* at FreeBSD.  When a new release is rolled out, this team is in charge for updating the port and committing it to the FreeBSD ports tree. Anybody is welcome to help the team by providing patches for [upgrading the ports](https://docs.freebsd.org/en/books/porters-handbook/book/#port-upgrading) following the [FreeBSD Porter's Handbook](https://docs.freebsd.org/en/books/porters-handbook/book/) instructions.
+
+Make sure the OpenSearch Team is aware of the release by droping a mail to opensearch@FreeBSD.org.
```

Comment:
I would just remove this or replace by saying "open an issue in xyz repo". Email addresses get notoriously spammy and you already mention it in the page you link.

Reply:
Removed!